cron_jobs/cronUniversityPayments.js

```diff
@@ -35,16 +41,53 @@ const getSummariesForUniversities = (allAppointmentsSummary) => {
   return universityContent
 }
 
-const run = async () => {
-  const { year, lastMonth } = date.getLastMonthAndYear(new Date())
-  console.log(`last month: ${lastMonth} and year ${year}`)
+module.exports.getSummariesForUniversities = getSummariesForUniversities
+
+async function formatSummaryEmail(summaryDate, psychologists) {
+  const html = await ejs.renderFile('./views/emails/summaryUniversity.ejs', {
+    lastMonth: summaryDate.lastMonth,
+    year: summaryDate.year,
+    psychologists: psychologists
+  });
+  return html
+}
+
+const sendMailToUniversities = async (summaryUniversities, summaryDate) => {
+  const allUniversities = await dbUniversities.getUniversities()
+
+  allUniversities.forEach(async (university) => {
+    const summaryUniversity = summaryUniversities[university.id]
+    if (summaryUniversity) {
+      const htmlFormated = await formatSummaryEmail(summaryDate, summaryUniversity);
+
+      if (!university.email) {
+        console.log(`Summary could not be send. ${university.name} doesn't have email.`)
```

Comment:
If the purpose is to exit it may even make sense to throw an error instead (`throw new Error("`Summary could not be send. ${university.name} should have an email.")`)

Reply:
Thank you for your feedback. Here it's not really an error, it's just an indication. It is possible that some universities do not have mail and i just want to log them. Maybe the `return;` is not really clean.